### PR TITLE
Solved the challenge

### PR DESCRIPTION
**Pull request by svetlivaksevski**

### CODE REVIEW

File: html-forms_Finding-a11y-errors/index.html [hunk 1]
```diff
@@ -19,14 +19,14 @@ <h1 class="title">Understanding A11y in Forms</h1>
...
           <input id="name-Input" type="text" name="name" />
-          <label aria-describedby="age-input">Age:</label>
+          <label for="age-input">Age:</label>
           <input id="age-Input" type="text" name="age" />
-          <label aria-describedby="color-input">Color you cannot see:</label>
+          <label for="color-input">Color you cannot see:</label>
           <input id="color-Input" type="color" name="color" />
           <textarea rows="7" placeholder="Leave us a message..."></textarea>
-          <button type="submit">Submit</button>
+          <button type="submit" id="submit">Submit</button>
```
Comment (Hainebach) [inline html-forms_Finding-a11y-errors/index.html:29]: I think you should add name as well with the same value as the id

File: html-forms_Finding-a11y-errors/index.html [hunk 2]
```diff
@@ -19,14 +19,14 @@ <h1 class="title">Understanding A11y in Forms</h1>
...
-          <label aria-describedby="name-input">Name:</label>
+          <label for="name-input">Name:</label>
           <input id="name-Input" type="text" name="name" />
-          <label aria-describedby="age-input">Age:</label>
+          <label for="age-input">Age:</label>
           <input id="age-Input" type="text" name="age" />
-          <label aria-describedby="color-input">Color you cannot see:</label>
+          <label for="color-input">Color you cannot see:</label>
           <input id="color-Input" type="color" name="color" />
           <textarea rows="7" placeholder="Leave us a message..."></textarea>
```
Comment (Hainebach) [inline html-forms_Finding-a11y-errors/index.html:28]: missing a label for the textarea

**Review by Hainebach — COMMENTED** [review]
had just a couple of comments, otherwise great

File: html-forms_Finding-a11y-errors/index.html (hunk 2 above)
Comment (svetlivaksevski) [reply html-forms_Finding-a11y-errors/index.html]: just added

**Pull request merged**